python/copr/v3/proxies/project_chroot.py

```diff
@@ -58,7 +58,9 @@ def edit(self, ownername, projectname, chrootname, additional_packages=None, add
         :param str chrootname:
         :param list additional_packages: buildroot packages for the chroot
         :param list additional_repos: buildroot additional additional_repos
-        :param list additional_modules: additional modules for the chroot
+        :param str additional_modules: comma separated list of modules that will
```

Comment:
I would prefer to take a list of modules instead of a comma-separated string because it makes much more sense for the `python-copr` user. If there is a bug in the API and it in fact expects a comma-separated string, then we could send `",".join(additional_modules)`.

Reply:
I believed that I will make a small change, but this is a bit complicated topic (the forms don't work ideally with the uploaded json structures).

Reply:
I mean, I eventually did a different fix - PTAL